src/backend/marsha/core/tests/test_views_lti_video.py

```diff
@@ -101,6 +101,37 @@ def test_views_lti_video_post_instructor(self, mock_get_consumer_site, mock_veri
         # signature)
         self.assertEqual(mock_verify.call_count, 1)
 
+    @mock.patch.object(LTI, "verify")
+    @mock.patch.object(LTI, "get_consumer_site")
+    @override_settings(EXTERNAL_JAVASCRIPT_SCRIPTS=["https://example.com/test.js"])
+    def test_views_lti_video_with_external_js_sources(
+        self, mock_get_consumer_site, mock_verify
+    ):
+        """Validate external js sources are added in the lti template."""
+        passport = ConsumerSiteLTIPassportFactory()
+        video = VideoFactory(
+            playlist__lti_id="course-v1:ufr+mathematics+00001",
+            playlist__consumer_site=passport.consumer_site,
+        )
+        data = {
+            "resource_link_id": video.lti_id,
+            "context_id": video.playlist.lti_id,
+            "roles": "administrator",
+            "oauth_consumer_key": passport.oauth_consumer_key,
+            "user_id": "56255f3807599c377bf0e5bf072359fd",
+            "launch_presentation_locale": "fr",
+        }
+
+        mock_get_consumer_site.return_value = passport.consumer_site
+        response = self.client.post("/lti/videos/{!s}".format(video.pk), data)
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, "<html>")
+        content = response.content.decode("utf-8")
+
+        self.assertIsNotNone(
+            re.search('<script src="https://example.com/test.js" >', content)
+        )
```

Comment:
IMO you should use `self.assertContains` instead: https://docs.djangoproject.com/en/2.2/topics/testing/tools/#django.test.SimpleTestCase.assertContains